docs/src/examples/new.md

```diff
@@ -0,0 +1,436 @@
+# Atmospheric cloud microphysics in an adiabatic air-parcel model
+
+This example constitutes a numerical implementation of a simple cloud physics model following :  
+**"An Elementary Parcel Model with Explicit Condensation and Supersaturation"**  
+by R.R. Rogers (1975).
+The purpose of this examples is twofold: to provide an atmospheric-physics example for the package, and to demonstrate a robust way of using `DifferentialEquations.jl` with the [`Unitful.jl`](https://juliaphysics.github.io/Unitful.jl/stable/) dimensional analysis package.
+This enables to programmatically represent physical units in the code and to check dimensionality corectnes in all arithmetic operations within physical formulae, while incurring zero overhead within numerical solution.
+
+The example code below reproduces all figures from the original paper by simulating the temporal evolution of supersaturation, droplet radius, temperature, and liquid water content in an ascending air parcel.
+
+---
+
+## Physical Background
+
+Clouds are suspensions of water droplets (and/or ice particles) in the air.
+Formation of a cloud is a thermodynamic process.
+To analyze it, we employ a simplified model based on the so-called air parcel framework, in which an adiabatically isolated "parcel" of air ascends  along a hydrostatic atmospheric pressure profile.
+Adiabatic cooling of air due to its expansion causes increase of relative humidity, which upon reaching supersaturation, triggers diffusional growth of droplets and concurrent latent heat release associated with the vapor-liquid phase transition.
```

Comment:
perhaps worth to clarify that RH=S, while here we refer to crossing the RH=1 threshold. Otherwise, it might be misleading to use "supersaturation" as a threshold here, and as a state variable above.